ArduPlane/quadplane.cpp

```diff
@@ -484,6 +484,13 @@ const AP_Param::GroupInfo QuadPlane::var_info2[] = {
     // @User: Standard
     AP_GROUPINFO("TAILSIT_GSCMIN", 18, QuadPlane, tailsitter.gain_scaling_min, 0.4),
 
+    // @Param: FWD_MANTHR_MAX
```

Comment:
the param name and description don't make it clear that this is related to tilting rotors

Reply:
it applies to all quadplanes

Reply:
ok. i think the description should reference the RCn_OPTION being set to create a fwd throttle option

Reply:
expand param desc and add arming check